Join custom field added only in zumstein project.
Compatibility with Amasty/Label 2.5.0.

diff --git a/etc/extension_attributes.xml b/etc/extension_attributes.xml
index b6ef06f..6f723d5 100644
--- a/etc/extension_attributes.xml
+++ b/etc/extension_attributes.xml
@@ -18,6 +18,9 @@
                 <field>color</field>
                 <field>text_color</field>
                 <field>text</field>
+                <!-- PATCH START -->
+                <field>image_2x</field>
+                <!-- PATCH END -->
             </join>
         </attribute>
     </extension_attributes>